python/pyarrow/tests/test_feather.py

```diff
@@ -599,6 +602,9 @@ def test_v2_set_chunksize():
 
 
 @pytest.mark.pandas
+@pytest.mark.lz4
+@pytest.mark.snappy
+@pytest.mark.zstd
```

Comment:
Could we expand this as:

```
pytest.mark.parametrize("codec", [
    pytest.param("lz4", marks=pytest.mark.lz4),
    pytest.param("snappy", marks=pytest.mark.snappy),
    pytest.param("zstd", marks=pytest.mark.zstd),
])
```

By letting pytest to iterate through the cases rather than doing it ourselves?

This would enable us to run this case just for the available plugins.